homeassistant/components/braviatv/coordinator.py

```diff
@@ -141,7 +142,13 @@ async def _async_update_data(self) -> None:
                 except BraviaAuthError as err:
                     raise ConfigEntryAuthFailed from err
 
-            power_status = await self.client.get_power_status()
+            # Retry up to 3 times in case request failed
+            async for attempt in AsyncRetrying(
+                reraise=True, stop=stop_after_attempt(3)
+            ):
+                with attempt:
+                    power_status = await self.client.get_power_status()
+
```

Comment:
Do we really have to add a new dependency to do this?
This can also cause a three-fold delay in changing the state when turning off the TV.

Reply:
Yeah perhaps it should. I noticed that tenacity was included as a dependency in the nibe_heatpump component, and then later moved into the backend library instead.

Reply:
I agree we should not introduce an external dependency just to retry, please implement it yourself.